ML-Frameworks/pytorch-aarch64/scripts/install_openblas.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# *******************************************************************************
+# Copyright 2025 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+# This script is based on the upstream PyTorch repository.
+# Reference: pytorch/pytorch
+# https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_openblas.sh
+# TODO: discard this and use `pytorch/.ci/docker/common/install_openblas.sh` to build OpenBLAS after version upgrade
+
+function apply-github-patch {
```

Comment:
rather than duplicate apply-github-patch, could we `source` from the existing build script or move these helper functions to `helper.sh` or `utils.sh` and source from there?